### ISSUE TITLE
Persistent layouts

### ISSUE THREAD

**Issue by Sinnbeck** [inertiajs/inertia-react#49]
Sorry for posting this here, but I didnt know here else to do so

Any plans on when to add persistent layouts to react? Is seems like a nice feature, and I can see it works perfectly when using vue :)

Edit: I made a quick "fix" to my current project, to get it working.
```
render(
    <InertiaApp
        children={({Component, props, key}) => {
            return <Layout children={<Component {...props} key={key} />} />
        }}
        initialPage={JSON.parse(app.dataset.page)}
        resolveComponent={name => import(`./Pages/${name}`).then(module => module.default)}
    />,
    app
);

**Comment by jamessessford** [comment]
Really looking forward to this feature - thanks for the workaround @Sinnbeck!

**Comment by diemah77** [comment]
Great workaround, thanks!

Any idea how to pass props to the layout from a page component? Something like a loading bar that is rendered in layout, but triggered from the page component.

**Comment by lovenick** [comment]
+1 to this. 

Just starting with Inertia and loving it. Might have time next week to dive into this feature.

**Comment by diemah77** [comment]
For now I'm using event bus to send data to the layout component.

**Comment by ecreeth** [comment]
I'm using Suspense API and I want to see how I can integrate the future concurrent mode API. See an example only with Suspense API [here](https://github.com/ecreeth/inertia-react-suspense)

**Comment by ecreeth** [comment]
This issue was fixed in #51
